atlas-operator/Chart.yaml

```
@@ -1,8 +1,8 @@
 ---
 apiVersion: v1
 name: atlas-operator
-version: 0.29.0
-appVersion: 0.12.0
+version: "0.30.0"
+appVersion: "0.13.0"
```

Its a nit, but I don't believe the quotes are necessary

```suggestion
version: 0.30.0
appVersion: 0.13.0
```

---

I agree, but I get a bit nervous around YAMLs loose typing and strings that look like floating point numbers.  If someone 
for some reason left off the trailing ".0" in the future then suddenly all hell would break loose and they'd have to be a YAML expert to figure out why.